packages/yew-macro/tests/function_component_attr/generic-pass.rs

```diff
@@ -58,20 +58,21 @@ fn comp1<T1, T2>(_props: &()) -> ::yew::Html {
     }
 }
 
-#[::yew::function_component(ConstGenerics)]
-fn const_generics<const N: ::std::primitive::i32>() -> ::yew::Html {
-    ::yew::html! {
-        <div>
-            { N }
-        </div>
-    }
-}
+// no longer possible?
+// #[::yew::function_component(ConstGenerics)]
+// fn const_generics<const N: ::std::primitive::i32>() -> ::yew::Html {
+//     ::yew::html! {
+//         <div>
+//             { N }
+//         </div>
+//     }
+// }
```

Comment:
What happened here? Seems like this PR broke something since the CI passes on master.

Reply:
const generics are only not possible in functions and the updated implementation now uses a function.